flexinput/build.gradle

```diff
@@ -38,9 +33,7 @@ dependencies {
     compile "com.android.support:support-v4:$supportLibraryVersion"
     compile "com.android.support.constraint:constraint-layout:$supportLibraryConstraintVersion"
 
-    compile "com.jakewharton:butterknife:8.4.0"
-    annotationProcessor "com.jakewharton:butterknife-compiler:8.4.0"
-
+    //noinspection GradleDynamicVersion
```

Comment:
? remove?

Reply:
It's a warning otherwise but if you don't mind it I can remove.

Reply:
I don't mind